### ISSUE TITLE
Have had to revert to 1.02 as 1.03 breaks my karma unit tests

### ISSUE THREAD

**Issue by iamkevla** [mtraynham/lodash-joins#7]
It cant evaluate any lodash functions 
   TypeError: 'undefined' is not a function (evaluating '_.chain(data)')
   undefined

**Comment by mtraynham** [comment]
Thanks for the report!  I see that it breaks with node.  Were you using it in the browser?

**Comment by mtraynham** [comment]
Weird, seems `_.size([])` doesn't work, but `_.default.size([])` does...  Probably an issue with the recent Babel upgrade.

**Comment by mtraynham** [comment]
Alrighty, did a little digging.  Seems the guys over at Babel have changed the way default exports work...  Not really sure why, but if you do `export default _`; you then have to `require('lodash-joins').default` (if using CommonJS).

Here's the Babel task for it:
https://phabricator.babeljs.io/T2212

Someone created a plugin for translating it back to the old way:
https://www.npmjs.com/package/babel-plugin-add-module-exports

I just followed suit and added a line to fix the export.  It's this libraries only public export, so there shouldn't be any other places that need to be updated.

I've tagged a new release (for bower) and v1.0.4 is up on npm, so let me know if it's still broke. :smile:  Thanks again!
